.github/workflows/on_pr_from_develop_to_master.yml

```
@@ -3,39 +3,119 @@ on:
+  mandatory-checks:
     if: github.head_ref == 'develop'
     runs-on: ubuntu-latest
     permissions:
       contents: read
-      packages: write
-      pull-requests: write
-
+    name: Mandatory File Changes
     steps:
       - name: Checkout the code from Repo
         uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)